package.json

```diff
@@ -6,14 +6,13 @@
   "publishConfig": {
     "access": "public"
   },
-  "main": "index.html",
```

Comment:
Otherwise this gets included in the published package, which is kind of useless.